test/infra/cchost.py

```diff
@@ -182,27 +182,22 @@ async def _start_process(self) -> None:
             cchost_env["FAKETIME_NO_CACHE"] = "1"
 
         LOG.debug("Starting cchost process...")
+        stdout_file = open(f"{self.workspace}/std.out", "w")
+        stderr_file = open(f"{self.workspace}/std.err", "w")
         process = await asyncio.create_subprocess_exec(
             self.binary,
             "--config",
             self.workspace / "config.json",
             cwd=self.workspace,
             start_new_session=True,
             stdin=subprocess.DEVNULL,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
```

Comment:
see https://docs.python.org/3/library/subprocess.html#subprocess.Popen.wait

> Note This will deadlock when using stdout=PIPE or stderr=PIPE and the child process generates enough output to a pipe such that it blocks waiting for the OS pipe buffer to accept more data. Use [Popen.communicate()](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate) when using pipes to avoid that.

Reply:
You mean that could be an issue before? Or is it a suggestion to some change?